ui/pages/home/home.component.js

```diff
@@ -778,15 +796,87 @@ export default class Home extends PureComponent {
     );
   };
 
+  getIsPrivacyToastRecent() {
+    const { newPrivacyPolicyToastShownDate } = this.props;
+
+    const currentDate = new Date();
+    const oneDayInMilliseconds = 24 * 60 * 60 * 1000;
+    const newPrivacyPolicyToastShownDateObj = new Date(
+      newPrivacyPolicyToastShownDate,
+    );
+    const toastWasShownLessThanADayAgo =
+      currentDate - newPrivacyPolicyToastShownDateObj < oneDayInMilliseconds;
+
+    return toastWasShownLessThanADayAgo;
+  }
+
+  renderToasts() {
+    const { t } = this.context;
+    const {
+      showSurveyToast,
+      showPrivacyPolicyToast,
+      setSurveyLinkLastClickedOrClosed,
+      setNewPrivacyPolicyToastClickedOrClosed,
+    } = this.props;
+    const { theme } = document.documentElement.dataset;
+
+    const isPrivacyToastRecent = this.getIsPrivacyToastRecent();
+
+    return showSurveyToast || showPrivacyPolicyToast ? (
+      <Box className="home__overlay-banners">
+        {showSurveyToast && (
+          <BannerBase
+            data-theme={
+              theme === ThemeType.light ? ThemeType.dark : ThemeType.light
+            }
+            backgroundColor={BackgroundColor.backgroundAlternative}
+            startAccessory={
+              <Icon name={IconName.Heart} color={IconColor.errorDefault} />
+            }
+            title={t('surveyTitle')}
+            actionButtonLabel={t('surveyConversion')}
+            actionButtonOnClick={() => {
+              global.platform.openTab({
+                url: 'https://www.getfeedback.com/r/Oczu1vP0',
+              });
+              setSurveyLinkLastClickedOrClosed(Date.now());
+            }}
+            onClose={() => {
+              setSurveyLinkLastClickedOrClosed(Date.now());
+            }}
+          />
+        )}
+        {showPrivacyPolicyToast && isPrivacyToastRecent && (
+          <BannerBase
+            data-theme="dark"
+            backgroundColor={BackgroundColor.backgroundAlternative}
+            startAccessory={
+              <Icon name={IconName.Info} color={IconColor.iconDefault} />
+            }
+            title={t('newPrivacyPolicyTitle')}
+            actionButtonLabel={t('newPrivacyPolicyActionButton')}
+            actionButtonOnClick={() => {
+              global.platform.openTab({
+                url: 'https://www.consensys.io/privacy-policy',
```

Comment:
This url needs to be used from a constant, look for this one in code `AUTO_DETECT_TOKEN_LEARN_MORE_LINK`